### PR TITLE
Document tuning DB comparison scripts

### PR DESCRIPTION
**Pull request by bernhardmgruber**
Mentions `sol.py` and `compare.py` in the tuning documentation.

### CODE REVIEW

File: docs/cub/benchmarking.rst [hunk 1]
```diff
@@ -229,6 +229,34 @@ run the benchmark regardless, and continue.
 The tuning infrastructure stores results in an SQLite database called `cccl_meta_bench.db` in the build directory.
 This database persists across tuning runs.
 If you interrupt the benchmark script and then launch it again, only missing benchmark variants will be run.
+
+
+Comparing results of multiple tuning databases
+--------------------------------------------------------------------------------
+
+Similarly to comparing JSON files from NVBench,
+benchmark results captured in a tuning database can be compared as well:
```
Comment (miscco) [inline docs/cub/benchmarking.rst:238]: Nitpick: I would shorten this to:
```suggestion
Benchmark results captured in a tuning database can be compared as well:
```

File: docs/cub/benchmarking.rst [hunk 2]
```diff
@@ -229,6 +229,34 @@ run the benchmark regardless, and continue.
...
+
+.. code-block:: bash
+
+    ../benchmarks/scripts/compare.py -o cccl_meta_bench1.db cccl_meta_bench2.db
+
+This will print a Markdown report showing the runtime differences and noise for each variant.
+
+Furthermore, you can plot the results from one or more tuning databases as bar chart or box plot (add `--box`):
```
Comment (miscco) [inline docs/cub/benchmarking.rst:246]: Question: Not a native speaker but shouldn't this either be `as a bar plot` or `as bar plots` because databases is plural?

File: docs/cub/benchmarking.rst [hunk 3]
```diff
@@ -229,6 +229,34 @@ run the benchmark regardless, and continue.
...
+.. code-block:: bash
+
+    ../benchmarks/scripts/sol.py -o cccl_meta_bench.db ...
+
+This is useful to display the current performance of CUB as captured in a single tuning database,
+or visually compare the performance of CUB across different tuning databases
+(from different points in time, on different GPUs, etc.).
+
+
+Dumping benchmark result from a tuning database
```
Comment (miscco) [inline docs/cub/benchmarking.rst:257]:
```suggestion
Dumping benchmark results from a tuning database
```

**Review by miscco — APPROVED** [review]

File: docs/cub/benchmarking.rst (hunk 2 above)
Comment (bernhardmgruber) [reply docs/cub/benchmarking.rst]: Agreed. Since we plot multiple databases as a single plot, let's add an article.
